doc/api/fs.md

```diff
@@ -2212,6 +2212,15 @@ fs.mkdir('/tmp/a/apple', { recursive: true }, (err) => {
 });
 ```
 
+On Windows, using `fs.mkdir()` on the root directory even with recursion will
```

Comment:
the nit picky editor in me wonders whether this should be:

```
using `fs.mkdir()` on the root directory, even with recursion, will
result in an error on Windows:
```

Reply:
My preference is for the fact that it's platform specific to be first but I'm willing to do whatever the consensus is.

Reply:
This maybe?

```markdown
On Windows, using `fs.mkdir()` on the root directory, even with recursion, will
result in an error:
```